setup.py

```diff
@@ -45,5 +51,13 @@
         "defconQt.representationFactories",
         "defconQt.util",
     ],
-    package_dir={"": "Lib"}
+    scripts=["trufont"],
```

Comment:
this console script will not work on Windows (you can't expect to have the bash shell there, and python script are executable by the python launcher only if they have the `.py` extension).

IMO, it would be better to use setuptools "entry points" to create platform-specific console scripts (.exe on Windows, or executable scripts with the correct shebang on Unix):

More info at https://pythonhosted.org/setuptools/setuptools.html#automatic-script-creation

setuptools has many advantages over plain distutils. It is the recommended method to distribute Python packages in the User Guide of the [PyPA](https://packaging.python.org/en/latest/current/) (Python PAckaging Authority). 

Together with the pip installer, setuptools is now included by default in all official Python.org installers and many other distributions.

OK, enough with the advertisement.